content/22.vaccines.md

```diff
@@ -247,6 +255,11 @@ Furthermore, the producers of Gam-COVID-Vac state that they can produce the vacc
 ### Protein Subunit Vaccines
 
 <!--Do we need to add this to the figure? Or does it fall in one of the other categories?-->
+Compared to the _inactivated whole virus vaccine_, these protein subunit vaccines isolate a single protein of the virus and use it to stimulate the immune system.
+These proteins, also referred to as antigens, are usually those located on the surface of the viral particle and therefore key targets of the immune system.
+These proteins are typically grown in yeast and then harvested.
+Because the vaccine contains only the purified antigens, the vaccine can only stimulate an antibody response [@doi:10.1007/978-1-4615-1823-5].
```

Comment:
perhaps add a clause like "in contrast to other types of vaccines that are able to stimulate helper T-cells" (and maybe B cells? I don't know enough about this)